cms/static/js/views/active_video_upload_list.js

```diff
@@ -204,27 +227,77 @@ define([
             },
 
             fileUploadDone: function(event, data) {
-                this.setStatus(data.cid, ActiveVideoUpload.STATUS_COMPLETED);
-                this.setProgress(data.cid, 1);
-                if (this.onFileUploadDone) {
-                    this.onFileUploadDone(this.collection);
-                    this.clearSuccessful();
-                }
+                var model = this.collection.get(data.cid),
+                    self = this;
+
+                this.readMessages([
+                    StringUtils.interpolate(
+                        gettext('Upload completed for video {fileName}'),
+                        {fileName: model.get('fileName')}
+                    )
+                ]);
+
+                this.sendStatusUpdate([
+                    {
+                        edxVideoId: model.get('videoId'),
+                        status: 'upload_completed',
+                        message: 'Uploaded completed'
+                    }
+                ]).done(function() {
+                    self.setStatus(data.cid, ActiveVideoUpload.STATUS_COMPLETED);
```

Comment:
@catong I am glad that issue is resolved for you but its weird that upload failed for a fully supported browser.

There is no `Pending` status. When an upload is completed successfully below will the statuses in order

`UPLOADED`               << after video is successfully uploaded to AWS S3
`IN PROGRESS`           << after video processing server start processing an uploaded video 
`READY`                       << after video is successfully processed by video processing server